scripts/cc-cli/src/commands/wizard.ts

```diff
@@ -222,14 +222,14 @@ function parseOptions(options: OptionValues) {
   }
 
   const rewardDestination = checkRewardDestination(
-    parseChoice(inputOrDefault(options.rewardDestination, "Staked"), [
+    parseChoiceOrExit(inputOrDefault(options.rewardDestination, "Staked"), [
       "Staked",
       "Stash",
       "Controller",
     ])
   );
 
-  const commission = parsePercentAsPerbill(
```

Comment:
Can we rename or unexport the `parsePercentAsPerbill()` function so that it's not accidentally used outside of tests and utils.ts ? It feels like a very easy error to make in the future, especially by someone who isn't as familiar with these functions as you are.